README.md

```diff
@@ -112,6 +112,20 @@ It starts etcd and simulator-server locally.
 
 To run the frontend, please see [README.md](web/README.md) on ./web dir.
 
+## [Beta] Existing cluster importing
+
+The simulator can import existing clusters.
+This allows for batch inclusion of resources from external clusters.
+It is enabled by an `EXTERNAL_IMPORT_ENABLED` environment variables is `1`.
+
+Please be careful (or avoid) using it in clusters with large amounts of resources.
+The simulator tries to import ALL resources related to scheduling from your cluster. 
```

Comment:
```suggestion
PLEASE AVOID using it in clusters with large amounts of resources.
The simulator will try to import ALL resources related to scheduling from your cluster and it may badly affect your cluster's api-server.
```

Reply:
Could you move this to around L116. It's so important one.